### PR TITLE
[NUI] Add IsWindowRotating()

### PR DESCRIPTION
**Pull request by sparrow74**
Add IsWindowRotationg function.
It is return with true when window is rotating.

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Interop/Interop.Window.cs [hunk 1]
```diff
@@ -265,6 +265,9 @@ internal static partial class Window
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_ExcludeInputRegion")]
             public static extern void ExcludeInputRegion(global::System.Runtime.InteropServices.HandleRef window, global::System.Runtime.InteropServices.HandleRef inputRegion);
+
+            [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_Window_IsWindowRotating")]
+            public static extern bool IsWindowRotating(global::System.Runtime.InteropServices.HandleRef window);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.Window.cs:270]: could you add this here?
`            [return: global::System.Runtime.InteropServices.MarshalAs(global::System.Runtime.InteropServices.UnmanagedType.U1)]`

this is for `bool` return type.
- reference : https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/internal/Interop/Interop.Actor.cs#L55

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs [hunk 2]
```diff
@@ -41,11 +43,33 @@ void Initialize()
...
+            tm = new Timer(100);
+            tm.Tick += Tm_Tick;
+            tm.Start();
+        }
+
+        private bool Tm_Tick(object source, Timer.TickEventArgs e)
+        {
+            bool rotating = mainWin.IsWindowRotating();
+            log.Fatal(tag, $"window is Rotating: {rotating}");
+            log.Fatal(tag, "window is Rotating: {}"+rotating);
```
Comment (hinohie) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs:71]: I think we dont need to reply this informations twice.

File: src/Tizen.NUI/src/internal/Interop/Interop.Window.cs (hunk 1 above)
Comment (sparrow74) [reply src/Tizen.NUI/src/internal/Interop/Interop.Window.cs:270]: I see, I will update with your comment.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs (hunk 2 above)
Comment (sparrow74) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/WindowTest.cs:71]: Ah..Thank you for detail review :)

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**